common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -34,4 +34,14 @@ trait ABTestSwitches {
     exposeClientSide = true,
   )
 
+  Switch(
+    ABTests,
+    "ab-deeply-read-article-footer",
+    "Test whether adding deeply read articles have negative impact on recirculation",
+    owners = Seq(Owner.withGithub("jlieb10")),
```

Comment:
```suggestion
    owners = Seq(Owner.withName("dotcom.platform")),
```

The other examples here and in other files do seem to favour Github ids for individuals but I think it would be preferable to keep this at a team level (in case of holidays, etc). I'm hoping my suggestion will work!